opencti-platform/opencti-front/src/private/components/common/stix_core_objects/StixCoreObjectQuickSubscriptionContent.tsx

```diff
@@ -141,19 +137,7 @@ StixCoreObjectQuickSubscriptionContentProps
   const [commitFieldPatch] = useMutation(triggerMutationFieldPatch);
   const [commitDeleteTrigger] = useMutation(TriggerPopoverDeletionMutation);
 
-  const searchInstanceTriggers = () => {
-    fetchQuery(stixCoreObjectQuickSubscriptionContentQuery, paginationOptions)
-      .toPromise()
-      .then((data) => {
-        setExistingInstanceTriggersEdges(
-          (data as StixCoreObjectQuickSubscriptionContentPaginationQuery$data)
-            ?.triggersKnowledge?.edges ?? [],
-        );
-      });
-  };
-
   const handleOpen = () => {
-    searchInstanceTriggers();
```

Comment:
this was to update the instance triggers list in case the user had deleted/updated one of triggers via the quick subscription edition panel

Reply:
but seems to work without it now that the 'pagination keys' are the same as Céline told me